opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -62,13 +62,18 @@
 """
 
 from contextlib import contextmanager
-import typing
+from typing import Callable
```

Comment:
IIRC we decided to import modules, i.e. `typing`, rather than importing individual members (not sure we documented it anywhere, btw).

Reply:
Please reconsider this.

```python
_TRACER_FACTORY: typing.Optional[
    typing.Callable[[typing.Type[Tracer]], typing.Optional[Tracer]]] = None
```
looks really terrible.

Reply:
One could also argue this importing a different class per line ;) I've never liked this in Java, for example. You can also always put each parameter in a new line.

Maybe @reyang happens to remember whether we had some style guidelines?

Reply:
> One could also argue the same with importing a different class per line ;)

And I'm one of them. We have to say "thank you" to `isort` for that.

Reply:
I'd like to propose change preferences for `isort` to make imports less ugly.

Reply:
This was documented in #36. I'm not entirely sure but if `isort` doesn't complain it should be fine? My opinion was that we definitely must not have `from x import y` with the same `x` multiple times (like this does), and we definitely must not have an `import x, y` (multiple *modules* in the same import). If we want to discuss other aspects, we should not do this here but reopen #36 or create a new issue. I agree that both possibilities that `isort` allows now are suboptimal (multiple `from import` for the same module is something I definitely don't want though).

Reply:
I've updated `isort` settings in c9821ec4d7ba7103aea040a8303ab9acf1c88733.

Reply:
Related PR: #109